docs/src/content/docs/ru/challenges/performance/12-scroll-cd.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 🟠 Оптимизация обнаружения изменений
+description: Задание 12 посвящено оптимизации количества циклов обнаружения изменений при прокрутке
+author: thomas-laforge
+contributors:
+  - tomalaforge
```

Comment:
you need to add your github handle instead of mine here and same for all pages you translated

Reply:
thank you, I changed it, though it's an authors of a task